apps/sms/views/inbox/js/startup.js

```diff
@@ -31,6 +33,7 @@
 
   exports.Startup = {
     init() {
+      ConversationClient.init(App.instanceId);
```

Comment:
So nobody uses this client yet, I added it since it doesn't harm, but can move this to follow-up if you guys want.